<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Select product</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="2" colspan="2">
           <widget class="QComboBox" name="cb_product">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pb_browse">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Product</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QLineEdit" name="le_products_path">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_operation">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Operation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="7" column="0" colspan="4">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Navigation</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QCheckBox" name="cb_pan_selection">
               <property name="text">
                <string>Pan to selection</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="cb_auto_flip">
               <property name="text">
                <string>Auto flip board if possible</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="pb_flip">
               <property name="text">
                <string>Top side</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="pb_view_fit">
               <property name="text">
                <string>View fit</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="cb_test">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Test name</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="4">
           <widget class="QTabWidget" name="tw_comm">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_4">
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <attribute name="title">
              <string>Components</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QTreeWidget" name="tw_components">
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="sortingEnabled">
                 <bool>true</bool>
                </property>
                <column>
                 <property name="text">
                  <string notr="true">1</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_5">
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <attribute name="title">
              <string>Testpoints</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QTreeWidget" name="tw_testpoints">
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="sortingEnabled">
                 <bool>true</bool>
                </property>
                <column>
                 <property name="text">
                  <string notr="true">1</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="4" column="0" colspan="4">
           <widget class="QTextEdit" name="te_test_info">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QPushButton" name="pb_save">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2" colspan="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pb_add_test">
            <property name="text">
             <string>Add test</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QCheckBox" name="cb_inc_all">
            <property name="text">
             <string>Include all pages</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pb_edit_test">
            <property name="text">
             <string>Edit test</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Information</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="1">
        <widget class="QPushButton" name="pb_next">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="8">
        <widget class="QTextEdit" name="te_picture_info">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pb_prev">
         <property name="text">
          <string>Previous</string>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="QPushButton" name="pb_add_pictures">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Add pictures</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="pb_open_folder">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Open folder</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="8">
        <widget class="QTextEdit" name="te_comp_tp_info">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="5">
        <widget class="QLabel" name="picture">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
